common/index.ts

```diff
@@ -164,3 +164,13 @@ export type Snowflake = `${bigint}`;
  * @internal
  */
 export type Permissions = `${bigint}`;
+
+/**
+ * https://discord.com/developers/docs/reference#message-formatting-formats
+ */
+export const FormattingPatterns = {
+	User: /<@!?(\d{17,19})>/,
```

Comment:
Should contain the 3 possibilities we'd want - User, user w nick and user with optional nick